src/main/command_executor.cpp

```diff
@@ -163,7 +171,8 @@ bool CommandExecutor::doCommandSingleton(Command* cmd)
     {
       getterCommands.emplace_back(new GetModelCommand());
     }
-    if (d_options.getDumpProofs() && res.isUnsat())
+    // for now only use this route for old proof
```

Comment:
Why do we need this?  I'd expect this behavior to not depend on new vs old proofs.